maths/shreedharacharya.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:math';
+//this function return a list of roots of a quadratic equation
+// [x1, x2] where x1 and x2 are roots of
+// aX^2 + bX + c = 0
+List shreedharacharya(double a,double b,double c) 
+{
+  double d = b*b - 4*a*c;
+  List A=[0,0];
+  if(d<0)
+  {
+    print('Imaginary roots');
+  }
+  else{
+    A[0] = (-b + pow(d,0.5))/(2*a);
+    A[1] = (-b - pow(d,0.5))/(2*a);
+    return A;
```

Comment:
You can move this `return` out of `if` statement.

Reply:
Sir this return statement is dealing with the condition for real roots ...only...

Reply:
Return type of this function is List, it's counter intuitive to return nothing. 
You can just return empty list when there are no real roots.